app/substrait_consumer_function_compatibility.py

```diff
@@ -0,0 +1,181 @@
+from typing import List, Optional
+
+import ibis
+import pandas as pd
+import streamlit as st
+from ibis import _
+
+st.set_page_config(layout="wide")
+
+
+def support_matrix_df():
+    return (
+        ibis.read_csv(
+            "https://raw.githubusercontent.com/richtia/consumer-testing/producer_support_matrix/app/consumer_results.csv"
```

Comment:
I don't think it's worth doing at this point but it might be interesting to consider using other data formats such as JSON or Protobuf.